src/test/java/org/folio/rest/api/TestBaseWithInventoryUtil.java

```diff
@@ -27,6 +27,11 @@
 
 
   protected static UUID createInstanceAndHolding(UUID holdingsPermanentLocationId)
+    throws ExecutionException, InterruptedException, MalformedURLException, TimeoutException {
+    return createInstanceAndHolding(holdingsPermanentLocationId, null);
+  }
+
+  protected static UUID createInstanceAndHolding(UUID holdingsPermanentLocationId, UUID holdingTempLocation)
```

Comment:
For consistency: Can you change `holdingTempLocation` to `holdingTempLocationId`?

Reply:
done